src/main/java/org/dkit/dermotlogue/App.java

```diff
@@ -123,6 +123,37 @@ private void demoPrimitiveTypeWrapping() {
         // Populate the ArrayList with 10 heights, print them out,
         // using a loop, calculate the average height and output it.
 
+        ArrayList<Double> heightList = new ArrayList<>();
+        heightList.add(1.67);
+        heightList.add(1.64);
+        heightList.add(1.57);
```

Comment:
Our static analysis engine detects that your pull request includes a change that **uses the rough value of Math.PI/2**. According to the [best practice](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cnt-rough-value-of-known-constant-found-cnt-rough-constant-value), it's recommended to use the predefined library constant for code clarity and better precision.